src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/CreateSDORespondent1NotificationHandler.java

```diff
@@ -55,34 +61,21 @@ public List<CaseEvent> handledEvents() {
     private CallbackResponse notifyRespondentSolicitor1SDOTriggered(CallbackParams callbackParams) {
         CaseData caseData = callbackParams.getCaseData();
 
-        notificationService.sendMail(
-            caseData.getRespondentSolicitor1EmailAddress(),
-            caseData.getCaseAccessCategory() == CaseCategory.SPEC_CLAIM
-                ? notificationsProperties.getSdoOrderedSpec()
-                : notificationsProperties.getSdoOrdered(),
-            addProperties(caseData),
-            String.format(REFERENCE_TEMPLATE, caseData.getLegacyCaseReference())
-        );
+        if (NOTIFY_RESPONDENT_SOLICITOR1_SDO_TRIGGERED.name().equals(callbackParams.getRequest().getEventId())) {
+            if (caseData.getRespondent1Represented() == YesOrNo.YES) {
```

Comment:
still i would create this as boolean method inside case data as i suggested earlier so we can reuse this flag elsewhere